zkevm-circuits/src/tx_circuit/sign_verify.rs

```diff
@@ -526,22 +553,15 @@ impl<F: Field> SignVerifyChip<F> {
                 .zip(powers_of_256.into_iter().rev())
                 .map(|(byte, coeff)| maingate::Term::Unassigned(*byte, coeff))
                 .collect_vec();
-            let (address, pk_hash_lo) = main_gate.decompose(ctx, &terms, F::ZERO, |_, _| Ok(()))?;
-
-            (
-                address,
-                pk_hash_lo
-                    .into_iter()
-                    .zip(pk_hash[12..].iter())
-                    .map(|(assigned, byte)| Term::assigned(assigned.cell(), *byte))
-                    .collect_vec(),
-            )
+            let (address,) = main_gate.decompose(ctx, &terms, F::ZERO, |_, _| Ok(()))?;
+
+            address
         };
         let is_address_zero = main_gate.is_zero(ctx, &address)?;
 
         // Ref. spec SignVerifyChip 3. Verify that the signed message in the ecdsa_chip
-        // with RLC encoding corresponds to msg_hash_rlc
-        let msg_hash_rlc = {
+        // with RLC encoding corresponds to msg_hash
+        let (msg_hash_lo, msg_hash_hi) = {
```

Comment:
This could be a single `msg_hash` of type `Word<_>`

Reply:
fixed [cf7e444](https://github.com/privacy-scaling-explorations/zkevm-circuits/pull/1418/commits/cf7e444991feb62ad14d93754d25989abd30ef57)